### PR TITLE
Update twitter logo

### PR DESCRIPTION
**Pull request by kohwdev**
Updated twitter logo to X. Issue #460.

### CODE REVIEW

File: .example.env
Comment (origranot) [inline .example.env:1]: Why did you remove it?

File: README.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ Happy Hacking !
 Simply copy and paste a URL into the provided area. Then click shorten URL! Your URL has now been shortened!
 
 <div align="center">
-<img src="docs/reduced-to.gif" width="600" height="254">
+<img src="assets/images/usage.gif" width="600" height="254">
```
Comment (origranot) [inline README.md:217]: Why did you changed it?
Make sure your branch is aligned with the master.

File: apps/frontend/src/components/home-buttons/twitter-button/twitter-button.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.twitter-x {
```
Comment (origranot) [inline apps/frontend/src/components/home-buttons/twitter-button/twitter-button.css:1]: Maybe it will be better to implement it within the svg with Tailwind?

**Review by origranot — CHANGES_REQUESTED** [review]
Hey @kohwdev,

Thank you for your contribution!
I left some comments, please address and fix the issues 🚀

File: .example.env
Comment (kohwdev) [reply .example.env:1]: Ah my bad, I just replaced the existing file to .env to run the app locally. 
I'm just starting out and trying to gain some experience by contributing, and I'm making mistakes here and there. I added the file again in the branch.

File: README.md (hunk 1 above)
Comment (kohwdev) [reply README.md:217]: It should align with the master branch now.

File: apps/frontend/src/components/home-buttons/twitter-button/twitter-button.css (hunk 2 above)
Comment (kohwdev) [reply apps/frontend/src/components/home-buttons/twitter-button/twitter-button.css]: Styled within the svg class with tailwind as advised.

File: README.md (hunk 1 above)
Comment (origranot) [reply README.md:217]: It's still not aligned with the master branch.

File: apps/frontend/src/components/home-buttons/twitter-button/twitter-button.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { component$ } from '@builder.io/qwik';
+import './twitter-button.css';
```
Comment (origranot) [inline apps/frontend/src/components/home-buttons/twitter-button/twitter-button.tsx:2]: Consider remove the unused import

File: apps/frontend/src/components/home-buttons/twitter-button/twitter-button.tsx [hunk 4]
```diff
@@ -10,17 +11,13 @@ export const TwitterButton = component$((props: TwitterButtonProps) => {
   return (
     <button
       type="button"
+      id="twitter-btn"
```
Comment (origranot) [inline apps/frontend/src/components/home-buttons/twitter-button/twitter-button.tsx:14]: Consider remove the id.

**Review by origranot — CHANGES_REQUESTED** [review]
Hey @kohwdev,

The branch is still not aligned with master branch, as the README file is outdated.
Please make sure to resolve the issues so I will be able to merge it.

Thank you for your time!
💯

**Comment by origranot** [comment]
@kohwdev Hey!

I just checked your changes and it looks like the button icon logo is not the same as the other icons.
imageimage

Another suggestion is changing most of the texts associated with "Twitter" to "X".
For example you can change the tooltip from Share on twitter to Share on X.
Also, there are some file names you can change.

Let me know what do you think 💯

**Review by fenilmodi00 — APPROVED** [review]

**Review by fenilmodi00 — APPROVED** [review]

**Comment by kohwdev** [comment]
> @kohwdev Hey!
> 
> I just checked your changes and it looks like the button icon logo is not the same as the other icons. imageimage
> 
> Another suggestion is changing most of the texts associated with "Twitter" to "X". For example you can change the tooltip from Share on twitter to Share on X. Also, there are some file names you can change.
> 
> Let me know what do you think 💯

ah I forgot to check the dark mode. I applied "currentColor" fill to the path element, color matches with other icons now.
